File: java/TuduManager/repository.postgres.sql2o/src/main/java/net/seymourpoler/tudumanager/repository/postgres/sql2o/todo/DeleteTodoRepository.java
package net.seymourpoler.tudumanager.repository.postgres.sql2o.todo;

import net.seymourpoler.tudumanager.domain.todo.delete.IDeleteTodoRepository;
import net.seymourpoler.tudumanager.repository.postgres.sql2o.DataBaseConnectionFactory;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

@Component
public class DeleteTodoRepository implements IDeleteTodoRepository {
    private final DataBaseConnectionFactory dataBaseConnectionFactory;

    @Autowired
    public DeleteTodoRepository(DataBaseConnectionFactory dataBaseConnectionFactory) {
        this.dataBaseConnectionFactory = dataBaseConnectionFactory;
    }

    @Override
    public void delete(Integer todoId) {
        final String sql = "DELETE FROM public.todos WHERE id = :id";
        try (var connection = dataBaseConnectionFactory.create()) {
            connection
                    .createQuery(sql)
                    .addParameter("id", todoId)
                    .executeUpdate();
        }
    }
}

File: dotnet/react.mvc-core.postgres.dapper/TuduManager/src/TuduManayer.Repository.Postgres.EntityFramework/Todo/Delete/DeleteTodoRepository.cs
using System.Linq;
using TuduManayer.Domain.Todo.Delete;

namespace TuduManayer.Repository.Postgres.EntityFramework.Todo.Delete
{
    public class DeleteTodoRepository : IDeleteTodoRepository
    {
        private readonly DataBaseContextFactory dataBaseContextFactory;

        public DeleteTodoRepository(DataBaseContextFactory dataBaseContextFactory)
        {
            this.dataBaseContextFactory = dataBaseContextFactory;
        }

        public void Delete(int todoId)
        {
            using var dbContext = dataBaseContextFactory.Create();
            var todo = dbContext.Todos.Single(x => x.id == todoId);
            dbContext.Todos.Remove(todo);
            dbContext.SaveChanges();
        }
    }
}